packages/js/ai/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Thanks for adding this, it will help get rid of some of the WC repo linting surprises when we do PRs. 👍